### PR TITLE
Add notice in RemoteServer dialog about account kinds and permissions

### LINKED ISSUE #483: No information about inability to log by Google account on Aggregate Server Configuration window.
#### Software versions 
Briefcase v1.10.x, Java v1.8.x, operating system Ubuntu, MacOS and Windows.

#### Problem description
Aggregate Server Configuration window doesn't contain information "Username cannot be a Google login; it must be an ODK Aggregate username with "Form Manager" permissions" as it used to before. 
It doesn't have to be the same message but it might be worth to inform users that they can't use their Google accounts. 

#### Steps to reproduce the problem
Click on Configure button.

### PR DESCRIPTION
**Pull request by ggalmazor**
Closes #483 

#### What has been done to verify that this works as intended?
Manually tested on Ubuntu
Verified that the correct permission is shown in pull and push tabs
Verified that the shown permissions are indeed what is needed for pulling/pushing forms

#### Why is this the best possible solution? Were any other approaches considered?
It's the narrowest change I could come with

#### Are there any risks to merging this code? If so, what are they?
Nope

#### Does this change require updates to documentation? If so, please file an issue at https://github.com/opendatakit/docs/issues/new and include the link below.
Nope.

### CODE REVIEW

**Comment by dcbriccetti** [comment]
I think these should use an enum and not Strings, but I don’t want that to hold up the beta.
`String requiredPermission`

What’s that derogatory joke? _string_ typing
http://wiki.c2.com/?StringlyTyped

File: src/org/opendatakit/briefcase/ui/reused/source/RemoteServerDialog.java [hunk 1]
```diff
@@ -68,9 +68,9 @@ private void showError(String error, String title) {
     );
   }
 
-  static RemoteServerDialog empty(RemoteServer.Test serverTester) {
+  static RemoteServerDialog empty(RemoteServer.Test serverTester, String requiredPermission) {
```
Comment (dcbriccetti) [inline src/org/opendatakit/briefcase/ui/reused/source/RemoteServerDialog.java]: I think these should use an enum and not Strings, but I don’t want that to hold up the beta.
`String requiredPermission`

What’s that derogatory joke? _string_ typing
http://wiki.c2.com/?StringlyTyped

**Comment by yanokwa** [comment]
**old**
old

**new**
new

The old is what I was expecting to see, basically a label. The new seems like a text box with grayed out text that cannot be edited. Any reason why we can't make it more of a label?

**Comment by ggalmazor** [comment]
I had trouble getting the line breaks on a label and I went for a text pane, and somehow I got to remove the ugly white background but now I see that it's not consistent across platforms.

I'll try to figure out how to use a label instead.

**Comment by ggalmazor** [comment]
I think I've fixed the text box. Will check on mac too

**Comment by kkrawczyk123** [comment]
Verified on Ubuntu, MacOS and Windows
@opendatakit-bot unlabel "needs testing"
@opendatakit-bot label "behavior verified"

**Pull request merged**